pom.xml

```diff
@@ -19,24 +19,67 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>org.eclipse.jnosql</groupId>
-        <artifactId>jnosql-parent</artifactId>
-        <version>1.0.0-b6-SNAPSHOT</version>
+        <groupId>org.eclipse.ee4j</groupId>
+        <artifactId>project</artifactId>
+        <version>1.0.7</version>
     </parent>
 
     <groupId>org.eclipse.jnosql.communication</groupId>
     <artifactId>jnosql-communication-driver</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
     <description>The Eclipse JNoSQL communication layer, Diana, implementations</description>
 
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+        <license>
+            <name>The Eclipse Public License v1.0</name>
+            <url>http://www.eclipse.org/legal/epl-v10.html</url>
+        </license>
+    </licenses>
+
     <scm>
-        <connection>scm:git:git://github.com/eclipse/jnosql-diana-driver.git</connection>
-        <developerConnection>scm:git:ssh://github.com:eclipse/jnosql-diana-driver.git</developerConnection>
-        <url>https://github.com/eclipse/jnosql-diana-driver</url>
+        <connection>scm:git:git://github.com/eclipse/jnosql-communication-driver.git</connection>
+        <developerConnection>scm:git:ssh://github.com:eclipse/jnosql-communication-driver.git</developerConnection>
+        <url>https://github.com/eclipse/jnosql-communication-driver</url>
     </scm>
 
     <properties>
+        <apache.rat.version>0.13</apache.rat.version>
```

Comment:
@dearrudam once both are JNoSQL I belive it is fine to use the same JNoSQL parent and don't duplicate those lines.

Reply:
@otaviojava yeah, you're right! I'm working on it! Thanks!

Reply:
@dearrudam please, let me know when you finished.

Reply:
@otaviojava please, go ahead!!